go.mod

```diff
@@ -3,51 +3,70 @@ module github.com/moov-io/paygate
 go 1.13
 
 require (
```

Comment:
Can we undo this change? Let's rebase this branch after https://github.com/moov-io/paygate/pull/597 is merged.

Reply:
No problem.